chance/p5js/Popup-Circles/index.md

```diff
@@ -3,7 +3,7 @@ layout: p5js
 author: Daniele Maselli
 author-url: https://github.com/DanieleMaselli 
 title: Popup Circles 
-tags: [P5.js, chance, repeat, random, Law of Similarity]
+tags: [P5.js, chance, random, form, Law of Connected Elements]
```

Comment:
I'm not talking about the content all the time. I'm talking about how you format them:  

```yaml
tags: 
  - P5.js
  - color
  - repeat
  - Law of Something
```